src/components/Sidebar/Sidebar.jsx

```diff
@@ -20,11 +41,46 @@ export const Sidebar = ({ children, className, footer, notifications, isFixed =
                 className,
             )}
         >
-            <div className={clsx("p-2 text-center xl:text-left", notifications ? null : "invisible")}>
-                <Counter className="text-sm">{notifications}</Counter>
+            <div className="flex p-2 sm:justify-center xl:justify-between">
+                <div
+                    className={clsx(
+                        "cursor-pointer sm:text-center",
+                        notifications?.announcements?.count ? null : "hidden",
+                    )}
+                >
+                    <Counter className="mr-2 text-sm" onClick={handleAnnounceClick} style={style}>
+                        <AnnounceIcon className="mr-1 sm:hidden xl:block" />
+                        {notifications?.announcements?.count}
+                    </Counter>
+                </div>
+
+                <div className={clsx("cursor-pointer sm:text-center", notifications?.notices?.count ? null : "hidden")}>
+                    <Counter className="text-sm" onClick={handleNotificationClick}>
+                        <BellIcon className="sm:hidden xl:block" />
+                        {notifications?.notices?.count}
+                    </Counter>
+                </div>
             </div>
 
-            <div className="mb-10 text-center">
+            <Drawer
+                classNames={{ left: "md:left-24 xl:left-50" }}
+                position="left"
+                title="Announcements"
```

Comment:
You said you're using notification props, but these hard-coded values are still tehre

Reply:
added title